Dockerfile

```diff
@@ -7,6 +7,10 @@ COPY ssl/*.conf /etc/nginx/conf.d/
 COPY ssl/server-certs/ /etc/ssl/
 COPY ./ /app
 
+# cronjob for updating CRLs
+RUN apt-get update && apt-get install cron -y
+RUN echo "* 23 * * * /app/script/sync-crls" >> /etc/cron.d/crls
```

Comment:
Does this need to run `pipenv run /app/script/sync-crls`?

Reply:
The `sync-crls` script itself does that: https://github.com/dod-ccpo/authnid/blob/crl-handling/script/sync-crls#L8